src/expressServer.js

```diff
@@ -1,5 +1,5 @@
 require('dotenv').config();
-
+require('express-async-errors');
```

Comment:
This line can be removed since we already require `express-async-errors` in this file on line 27

Reply:
Deleted 👍, commit 690de2ede2bcf1ff5d191ae45108c75b09ce9520